net/dhtd/Makefile

```
@@ -0,0 +1,44 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dhtd
+PKG_VERSION:=0.2.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://codeload.github.com/mwarning/dhtd/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE:=dhtd-$(PKG_VERSION).tar.gz
+PKG_HASH:=044dcdcf760929daba4661bf9eac461d822ad94493aa4d9dc9623625deca72a2
+
+PKG_MAINTAINER:=Moritz Warning <moritzwarning@web.de>
+PKG_LICENSE:=MIT
+
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/dhtd
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=DHT Daemon
+	SUBMENU:=IP Addresses and Names
+	URL:=https://github.com/mwarning/dhtd
+	MENU:=1
+endef
+
+define Package/dhtd/description
+	Standalone BitTorrent DHT daemon. Lookup and announce
+	hash identifiers via command line interface.
+endef
```

```suggestion
define Package/dhtd
  SECTION:=net
  CATEGORY:=Network
  SUBMENU:=IP Addresses and Names
  TITLE:=DHT Daemon
  URL:=https://github.com/mwarning/dhtd
endef

define Package/dhtd/description
  Standalone BitTorrent DHT daemon. Lookup and announce
  hash identifiers via command line interface.
endef
```

use 2 spaces instead of tab here